packages/core/src/Stores/portfolio-store.js

```diff
@@ -587,4 +588,8 @@ export default class PortfolioStore extends BaseStore {
     get is_multiplier() {
         return this.contract_type === 'multiplier';
     }
+
+    get is_turbos() {
```

Comment:
@maryia-deriv Please add the type in `packages/stores/types.ts` 🙏🏻

Reply:
Will do🙇🏼‍♀️

Reply:
done👌🏻